moped-database/migrations/1700515730257_fix_delete_component_bug/up.sql

```diff
@@ -0,0 +1,210 @@
+-- latest version 1699978012000_change_contractor_column_name
+DROP VIEW project_list_view CASCADE;
```

Comment:
This cascade looks to be dropping `component_arcgis_online_view` so it seems like that view needs to be added back in this PR. I feel a little out of the loop on the dependencies here.

Reply:
you are right and now im wondering how  i didnt get the same error. fixing now

Reply:
okay i fixed it now and also have figured out how i missed the error -- i was applying the migration by hand and going up and down and testing that way. not applying the metadata, or spinning it up completely from scratch. good reminder for me to do that before requesting review!